draft-ietf-tls-esni.md

```diff
@@ -494,15 +492,17 @@ encapsulated key, context, HRR key (see {{client-hrr}}), and payload as:
 
 ~~~
     pkR = Deserialize(ECHConfig.public_key)
-    enc, context = SetupBaseS(pkR, "tls13 ech")
+    enc, context = SetupBaseS(pkR, "tls13 ech\0" + ECHConfig)
```

Comment:
I wonder if `\0` is the right way to right the null byte. (RFC8446 doesn't encode 0x00 in a string at all: see [RFC8446, 4.4.3](https://tools.ietf.org/html/rfc8446#section-4.4.3).) I'm fine with it as-is, since I think it's pretty clear what you mean ... I just wanted flag it in case there's a "correct" way to write this.